testsuite/tests/kuadrant/authorino/identity/api_key/test_reconciliation.py

```diff
@@ -20,7 +20,7 @@ def auth(api_key):
 @pytest.fixture(scope="module")
 def authorization(authorization, module_label):
     """Creates AuthConfig with API key identity"""
-    authorization.identity.api_key("api_key", match_label=module_label)
+    authorization.identity.add_api_key("api_key", selector=Selector(matchLabels={"group": module_label}))
```

Comment:
Wouldn't this be correct? 
Change `api_key` for `module_label` and:
```suggestion
    authorization.identity.add_api_key("api_key", selector=api_key.selector)
```

Reply:
This is because that test requires configuration for an API Key that does not exist yet (and will be created in the actual test)